static/js/admin/src/action/RunEnhancementsBatchAction.js

```diff
@@ -30,8 +30,12 @@ define('RunEnhancementsBatchAction', [ 'jquery', 'AbstractBatchAction', "tpl!../
 		} else {
 			title = "Run enhancements on " + this.targets.length + " objects";
 		}
-		
-		var form = runEnhancementsTemplate();
+
+		var targetIds = "";
+		for (var index in this.targets) {
+			targetIds += this.targets[index].getPid() + "\n";
+		}
+		var form = runEnhancementsTemplate({ targetIds : targetIds });
```

Comment:
where is `runEnhancementsTemplate` defined? how does it accept this new argument?

Reply:
it's defined in static/templates/admin/runEnhancementsForm.html, which is imported into this file in the first line:
https://github.com/UNC-Libraries/box-c/blob/da0248538409c14e4ef93f9ea22189a41d231a3c/static/js/admin/src/action/RunEnhancementsBatchAction.js#L1
The template library that we're using can take an object as an argument, and it exposes the values in the object as parameters in the template